config/auth_default.go

```diff
@@ -11,6 +11,7 @@ import (
 
 var (
 	authProviders = []CredentialsProvider{
+		MetadataServiceCredentials{},
```

Comment:
Please move it to the bottom of the list, we don't assume it'll be triggered for top-10% of the products built on top of Go SDK

Reply:
Can you move it to the bottom of the list , or at least below the `BricksCliCredentials`, because it's very risky to execute it first in the chain.